Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ruby:2.6
+
+COPY . /usr/local/src
+WORKDIR /usr/local/src
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends build-essential
+
+# 2.6-slim-stretch comes with bundler 1.17 installed, but we need >2.0.
+RUN gem uninstall -x bundler && gem install bundler -v 2.0.1
```

Comment:
I also had to add `gem "ffi", "< 1.17.0"` ([github issue](https://github.com/ffi/ffi/issues/1103)) to `Gemfile` get the bundles to install.